packages/dashboard/src/components/widgets/widgetActions.css

```diff
@@ -0,0 +1,7 @@
+.widget-actions-container {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  top: -20px;
+  z-index: 999;
```

Comment:
Is there a reason to chose `999` here? Maybe we can add a comment to explain which elements it overlaps with to need this z-index value.

Reply:
<img width="422" alt="Screenshot 2024-02-17 at 3 44 38 AM" src="https://github.com/awslabs/iot-app-kit/assets/139960352/c9768516-e514-48d3-a1ff-58acfb52c411">

Added comment for the z-index, if we give z-index value like '99' then it shows like the above screenshot